extensions/linz/examples/item.json

```diff
@@ -24,7 +24,10 @@
   "properties": {
     "datetime": "2020-12-11T22:38:32Z",
     "proj:epsg": 123,
-    "version": "1.0.0"
+    "version": "1.0.0",
+    "processing:software": {
+      "pystac": "v1.2.0"
```

Comment:
I know this is an example and will pass the tests but is this likely to be pystac in the real world? That is the software used for the metadata creation rather than the imagery processing pipeline. This is our spec for the Topo Processor processing:software field: `"Topo Processor":"<hash of commit used to process data>"`

Reply:
As discussed, pystac is probably a bad example since it's used to process the metadata rather than the assets. I've opted to keep it for now.